arithmetization/src/test/java/net/consensys/linea/zktracer/containers/StackedListTests.java

```diff
@@ -52,22 +49,6 @@ protected int computeLineCount() {
     }
   }
 
-  @Test
-  void testAddedToFront() {
-    final StackedList<IntegerModuleOperation> state = new StackedList<>();
-
-    state.enter();
-    state.add(new IntegerModuleOperation(1));
-    assertThat(state.lineCount()).isEqualTo(1);
-
-    state.enter();
-    state.add(new IntegerModuleOperation(3));
-    assertThat(state.lineCount()).isEqualTo(4);
-
-    state.pop();
-    assertThat(state.lineCount()).isEqualTo(1);
-  }
-
```

Comment:
What was the purpose of that test @letypequividelespoubelles ?

Reply:
a very usefull test deleted by mistake